background/token-manager.js

```diff
@@ -177,6 +185,7 @@ const tokenMan = (() => {
         grant_type: 'authorization_code',
         client_id: provider.clientId,
         redirect_uri: query.redirect_uri,
+        state,
```

Comment:
Or I've missed something, but it should send the state to the `access_token` endpoint. 🤷🏽‍♀️